substrate/client/chain-spec/src/lib.rs

```diff
@@ -40,14 +250,10 @@
 //!
 //! pub type MyChainSpec<G> = GenericChainSpec<G, MyExtension>;
 //! ```
-//!
-//! Some parameters may require different values depending on the
-//! current blockchain height (a.k.a. forks). You can use `ChainSpecGroup`
-//! macro and provided [`Forks`](./struct.Forks.html) structure to put
-//! such parameters to your chain spec.
-//! This will allow to override a single parameter starting at specific
-//! block number.
-//!
+//! Some parameters may require different values depending on the current blockchain height (a.k.a.
```

Comment:
Isn't this paragraph content overlapping with the `README.md`?

If there are two versions we have two things to keep in sync. Maybe we should remove this things from README?